### PR TITLE
Feat:Added login and products api integration

### PR DESCRIPTION
**Pull request by scjuly19**

### CODE REVIEW

File: src/pages/Wishlist.js [hunk 1]
```diff
@@ -26,32 +20,46 @@ export default function Wishlist() {
...
+        )}
+        {data.length > 0 ? (
+          <section id="wishlist" className="wishlist-wrapper">
+            {data.map((item) => (
+              <Card
+                item={item}
+                key={item.id}
+                isWishlist={true}
+                // onWishlistClick={handleRemoveWishlist(item.id)}
+                // onAddToCartClick={handleAddToCart(item)}
```
Comment (samyakshah3008) [inline src/pages/Wishlist.js:45]: Can remove this commented code

**Review by mdarif6 — APPROVED** [review]
-import exporting function is looking nice.
-folder structure is also looking good.
-I think you have done multiple changes in one go due to which code reading is taking more time, rest is okay.

**Review by samyakshah3008 — APPROVED** [review]
Good use of semantic html tags with clean code and proper folder structure 

Suggestion: You can also provide description as preview for PR 
Can give live link as well for better review 

Rest is all good ✅

File: src/pages/Wishlist.js (hunk 1 above)
Comment (scjuly19) [reply src/pages/Wishlist.js:45]: Actually I have kept it because it will be used once I add the other logic

**Comment by scjuly19** [comment]
@mdarif6 Thank you for your comments. I am aware that changes are a bit more for one PR , I will try to keep them less for the next one.

**Comment by scjuly19** [comment]
Here's the live link of the site: https://fluff-fy-tails.netlify.app/

**Pull request merged**